language/move-native/src/panic.rs

```diff
@@ -0,0 +1,7 @@
+use core::panic::PanicInfo;
+
+#[panic_handler]
+fn panic(_info: &PanicInfo) -> ! {
+    // panic inside polkavm program should call abort or smth similar
```

Comment:
Maybe add a log message here before going into an endless loop. Could safe someone a lot of time one-day.

Reply:
maybe not yet drop the entire crate, but you could remove all changes to `move-native` in this PR